plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/messages.properties

```diff
@@ -16,4 +16,8 @@ AppEngineJavaComponentsNotInstalled=App Engine Java components not installed
 JavaNotInstalled=Could not find JDK
 NoSuchDirectory=Directory {0} does not exist
 FileNotDirectory={0} is a file, not a directory
-UpdateSdk=Update
\ No newline at end of file
+UpdateSdk=Update
+CloudSdkUpdateNotificationTitle=Google Cloud SDK Update Available
+CloudSdkUpdateNotificationMessage=\
+ <a href="update">Update now</a>, or update later via "Google Cloud Tools" preferences.\n\
```

Comment:
I think it's clearer to say "update later in the "Google Cloud Tools" preferences page". "Via ... preferences" is a bit ambiguous.

Reply:
perhaps "in the Google Cloud Tools preferences"?

Reply:
_or update later from the Google Cloud Tools preference page_?